docs/admin/install.md

```diff
@@ -65,3 +89,14 @@ The plugin behavior can be controlled with the following list of settings:
 | ------------------- | ------- | ------- | ---------------------------------------------------------- |
 | `hide_example_jobs` | `True`  | `False` | A boolean to represent whether or display the example job. |
 
+## Integrations Configuration
+
+The `nautobot-ssot` package includes multiple integrations. Each requires extra dependencies defined in `pyproject.toml`. See the [installation guide](#installation-guide) for instructions on installing these dependencies.
```

Comment:
This link seems wrong and is the dependencies info missing?

Reply:
Fixed